api/tests/unit/domain/models/CertificationIssueReport_test.js

```diff
@@ -20,7 +20,7 @@ describe('Unit | Domain | Models | CertificationIssueReport', () => {
       };
 
       it('should not throw any error', () => {
-        expect(() => CertificationIssueReport.new(certificationIssueReportDTO)).to.not.throw;
+        expect(() => CertificationIssueReport.new(certificationIssueReportDTO)).to.not.throw();
       });
```

Comment:
Pour les `'should not throw an error'`, je propose de décrire ce qu'on veut (plutôt que ce qu'on ne veut pas), ça me paraît plus clair. Par exemple :

```javascript
      it('should create a CertificationIssueReport', () => {
        expect(CertificationIssueReport.new(certificationIssueReportDTO)).to.be.an.instanceOf(CertificationIssueReport);
      });
```

La notion d'error et de throw est décrite dans les 'should throw', pas besoin de l'ajouter ici à mon avis.